src/clients/InventoryClient.ts

```diff
@@ -225,29 +225,11 @@ export class InventoryClient {
     let cumulativeVirtualBalanceWithShortfall = cumulativeVirtualBalance.sub(chainShortfall);
     let chainVirtualBalanceWithShortfallPostRelay = chainVirtualBalanceWithShortfall.sub(outputAmount);
 
-    const startTime = Date.now();
-    let totalRefundsPerChain: { [chainId: string]: BigNumber } = {};
-    try {
-      // Consider any refunds from executed and to-be executed bundles.
-      totalRefundsPerChain = await this.getBundleRefunds(l1Token);
-    } catch (e) {
-      this.log("Failed to get bundle refunds, defaulting refund chain to hub chain", {
-        l1Token,
-        originChainId,
-        destinationChainId,
-        error: e,
-      });
-      // Fallback to ignoring bundle refunds if calculating bundle refunds goes wrong.
-      // This would create issues if there are relatively a lot of upcoming relayer refunds that would affect
-      // the relayer's repayment chain of choice.
-      totalRefundsPerChain = Object.fromEntries(
-        this.getEnabledChains().map((chainId) => {
-          return [chainId, bnZero];
-        })
-      );
-    }
-
-    this.log(`Time taken to get bundle refunds: ${Math.floor(Date.now() - startTime) / 1000}s`, {
+    const startTime = performance.now();
+    // Consider any refunds from executed and to-be executed bundles. If bundle data client doesn't return in
+    // time, return an object with zero refunds for all chains.
+    const totalRefundsPerChain: { [chainId: string]: BigNumber } = await this.getBundleRefunds(l1Token);
+    this.log(`Time taken to get bundle refunds: ${Math.floor(performance.now() - startTime) / 1000}s`, {
```

Comment:
Minor, but `Math.floor()` has no effect on any of these calculations because performance.now() and startTime are both at ms precision. `Math.round()` is probably also a better choice because it'll round to the nearest integer, rather than unconditionally round downwards.

```suggestion
    this.log(`Time taken to get bundle refunds: ${Math.round((performance.now() - startTime) / 1000)}s`, {
```